### ISSUE TITLE
No handling of null values in filter

### ISSUE THREAD

**Issue by ErisDS** [scoutforpets/bookshelf-jsonapi-params#31]
Pretty sure the issue is with this repo, rather than https://github.com/kideh88/node-jsonapi-query-parser.

Try to filter for null values doesn't work. If I try to set something like `filter[deleted_at]=null` in the query string, I get an empty result set. 

If I try to set the filter programmatically before calling bookshelf-jsonapi-params, e.g.

```
options.filter = {
    deleted_at: null
};
```

I get this error:

```
TypeError: Cannot read property 'toString' of null
    at /Users/hannah/Ghost/org/Daisy.js/node_modules/express-bookshelf-jsonapi/node_modules/bookshelf-jsonapi-params/dist/index.js:385:50
    at /Users/hannah/Ghost/org/Daisy.js/node_modules/express-bookshelf-jsonapi/node_modules/bookshelf-jsonapi-params/node_modules/lodash/lodash.js:4411:15
    at baseForOwn (/Users/hannah/Ghost/org/Daisy.js/node_modules/express-bookshelf-jsonapi/node_modules/bookshelf-jsonapi-params/node_modules/lodash/lodash.js:2654:24)
    at /Users/hannah/Ghost/org/Daisy.js/node_modules/express-bookshelf-jsonapi/node_modules/bookshelf-jsonapi-params/node_modules/lodash/lodash.js:4380:18
    at forEach (/Users/hannah/Ghost/org/Daisy.js/node_modules/express-bookshelf-jsonapi/node_modules/bookshelf-jsonapi-params/node_modules/lodash/lodash.js:8634:14)
    at QueryBuilder.<anonymous> (/Users/hannah/Ghost/org/Daisy.js/node_modules/express-bookshelf-jsonapi/node_modules/bookshelf-jsonapi-params/dist/index.js:310:41)
    at Object.query (/Users/hannah/Ghost/org/Daisy.js/node_modules/bookshelf/lib/helpers.js:56:14)
    at query (/Users/hannah/Ghost/org/Daisy.js/node_modules/bookshelf/lib/model.js:1206:30)
```

Should this work? is there a special syntax that I should use?

**Comment by jamesdixon** [comment]
@ErisDS I haven't personally tried it, but theoretically it should work.

I don't have enough time to dig into the code, but if you're able to find the issue, we'd love a PR!

**Comment by alechirsch** [comment]
@ErisDS I just ran into this problem and am working on a fix

**Comment by alechirsch** [comment]
@ErisDS @jamesdixon I fixed this in #25. I fixed it in that pull request because there would be merge conflicts otherwise.
